features/logging/logging_acceptance.feature

```diff
@@ -0,0 +1,152 @@
+    Given evaluation of `user.cached_tokens.first` is stored in the :user_token clipboard
+    And I use the "openshift-logging" project
+    And I wait for the project "<%= cb.proj.name %>" logs to appear in the ES pod
+    When I perform the HTTP request on the ES pod with labels "es-node-master=true":
+      | relative_url | app*/_count?format=JSON' -d '{"query": {"match": {"kubernetes.namespace_name": "<%= cb.proj.name %>"}}} |
+      | op           | GET                                                                                                     |
+      | token        | <%= cb.user_token %>                                                                                    |
+    Then the step should succeed
+    And the expression should be true> @result[:parsed]['count'] > 0
+    When I perform the HTTP request on the ES pod with labels "es-node-master=true":
+      | relative_url | infra*/_count?format=JSON |
+      | op           | GET                       |
+      | token        | <%= cb.user_token %>      |
+    Then the step should succeed
+    And the expression should be true> @result[:parsed]['count'] > 0
+    When I perform the HTTP request on the ES pod with labels "es-node-master=true":
+      | relative_url | app*/_count?format=JSON |
+      | op           | GET                     |
+      | token        | <%= cb.user_token %>    |
+    Then the step should succeed
+    And the expression should be true> @result[:parsed]['count'] > 0
+# Cronjob
+    Given the expression should be true> cluster_logging('instance').management_state == "Managed"
+    And the expression should be true> elasticsearch('elasticsearch').management_state == "Managed"
+    Given evaluation of `cron_job('curator').schedule` is stored in the :curator_schedule_1 clipboard
+    Then the expression should be true> cb.curator_schedule_1 == cluster_logging('instance').curation_schedule
+    When I run the :patch client command with:
+      | resource      | clusterlogging                                                    |
+      | resource_name | instance                                                          |
+      | p             | {"spec": {"curation": {"curator": {"schedule": "*/15 * * * *"}}}} |
+      | type          | merge                                                             |
+    Then the step should succeed
+    And the expression should be true> cluster_logging('instance').curation_schedule == "*/15 * * * *"
+    And I wait up to 180 seconds for the steps to pass:
+    """
+    Given the expression should be true> cron_job('curator').schedule(cached: false, quiet: true) == "*/15 * * * *"
+    """
+    When I run the :patch client command with:
+      | resource      | cronjob                                 |
+      | resource_name | curator                                 |
+      | p             | {"spec": {"schedule": "*/20 * * * *" }} |
+    Then the step should succeed
+    Given 60 seconds have passed
```

Comment:
We can use 'wait up to' to replace hard wait time.

Reply:
@liangxia I have updated the code as suggested, thanks.